minetest/minetest
# Add support for single and dead-end raillike textures

Fixes #2386.

This adds optional support for two additional raillike textures: single (no connection) and dead-end (one connection). These can be specified in the 5th and 6th tile, respectively. Each of them will fall back to the straight texture, if missing.

https://satoshiupload.com/images/WoVW8jNIS0.png

## To do

This PR is ready for review.

## How to test

Use DevTest and play around with the new Railike test nodes. See the DevTest code for details.
